buildtest/cli/buildspec.py

```diff
@@ -973,14 +971,14 @@ def print_raw_filter_fields(self):
         method implements command ``buildtest buildspec find --filterfields``
         """
         for field in self.filter_fields:
-            console.print(field)
+            console.print(f"[{self.color}]{field}")
```

Comment:
i think this statement you can do `console.print(field,style=self.color)`.

The only time you may want to wrap this around fstring is when you need to color part of a word like `console.print("[red]red text[/red] and normal text")`. Sometimes data can't be represented with `console.print(data)` where data is not printed properly and therefore one needs to do `console.print(f"{data}")`. This is true for some data-types like class instances

Reply:
@shahzebsiddiqui Okay got it, fixed this !